### DISCUSSION TITLE
Issue with skinGui and other applications

### DISCUSSION THREAD

**Discussion by justdiandes** [robotology/community#708] [Q&A]
Hi everyone! 

So as mentioned in the issue #707, we tried to update the robotology dependencies in both a computer and the icub-head. From there we faced problems with the other Applications. We have some issue with the emotions one, that we fixed earlier but after an upgrade the module stops to work, so the applications starts but the icub doesn't changes the expression, but we think is due to the version of the XML.

In the same way, the module related to the skinGui doesn't work either, in the sense that all the modules run good but the gui doesn't show any interactions when we touch the sensors. We'd like to know if there's any way to check the version of every XML for the applications that are installed in the yarpmanager, because even when the installation of the new version of robotology, the yarp modules didn't update.

**Comment by Nicogene** [comment]
> We'd like to know if there's any way to check the version of every XML for the applications that are installed in the yarpmanager, because even when the installation of the new version of robotology, the yarp modules didn't update.

Unfortunately, there is no versioning for the yarpmanager application xml, usually, they are created when the robot is prepared to be shipped (cc @AntonioConsilvio). 

In general, we do not make major changes to those xml, so probably the problems you are experiencing are due to something else.

> In the same way, the module related to the skinGui doesn't work either, in the sense that all the modules run good but the gui doesn't show any interactions when we touch the sensors.

So the modules run fine? Have you connected all the ports?

The log of the executables involved in this App can help in understanding which is the problem you are having.
